tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -198,11 +222,16 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
       |}
       |""".stripMargin,
     """|given (commit: '')
+       |given_FromString_Int - scala.util.CommandLineParser.FromString (commit: '')
```

Comment:
How did these get here? :thinking: Those are member and we normally don't show those. Should we create a follow up?

Reply:
https://github.com/scalameta/metals/issues/5992